app.js

```diff
@@ -18,8 +18,8 @@ app.get("/aboutus", async (request, response) => {
 app.get("/movies", async (request, response) => {
   response.render("movies");
 });
-app.get("/404", async (request, response) => {
-  response.render("404");
+app.get("/404", async (req, res) => {
+  res.status(404).render("404");

```

Comment:
A separate /404 route is not a great idea. There are some UX problems, but on a more philosophical level, if there exists a resource called "404" and the HTTP client requests it, is it really correct to say that the status is 404? That status means "not found", but the 404 route was found!

I understand what you're trying to do, but the correct way to respond with a 404 is not this (see next comment for better solution).